src/main/java/Consolidator.java

```diff
@@ -154,6 +154,14 @@ protected void processLine(String line) {
         			currentBlock.requireNonEmptySource = true;
         			return;
         		}
+        		if(row[1].equals("treatFormulaAsBlank")) {
+        			currentBlock.treatFormulaAsBlank = true;
+        			return;
+        		}
+        		if(row[1].equals("autoSkipOnError")) {
```

Comment:
every config var is on a new line, so it's always the second element, I'm fine with changing it to an else if structure